### PR TITLE
Jenkins e2e: Export KUBE_ENABLE_DAEMONSETS

### PR DESCRIPTION
**Pull request by justinsb**
We manipulate KUBE_ENABLE_DAEMONSETS in the script, but unless we export
it, it won't actually be passed down to child processes.

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -992,6 +992,7 @@ export E2E_MIN_STARTUP_PODS=${E2E_MIN_STARTUP_PODS:-}
 export KUBE_ENABLE_CLUSTER_MONITORING=${ENABLE_CLUSTER_MONITORING:-}
 export KUBE_ENABLE_CLUSTER_REGISTRY=${ENABLE_CLUSTER_REGISTRY:-}
 export KUBE_ENABLE_HORIZONTAL_POD_AUTOSCALER=${ENABLE_HORIZONTAL_POD_AUTOSCALER:-}
+export KUBE_ENABLE_DAEMONSETS=${ENABLE_DAEMONSETS:-}
```
Comment (lavalamp) [inline hack/jenkins/e2e.sh]: Name change is intentional, correct?

**Comment by lavalamp** [comment]
LGTM

**Comment by justinsb** [comment]
Ugh - thanks for spotting.  Looks like there's also a mix of ENABLE_DEPLOYMENTS & KUBE_ENABLE_DEPLOYMENTS in this file too.  Removed LGTM while I try to make sense of it...

**Comment by lavalamp** [comment]
Thanks for looking into it!

**Comment by bgrant0607** [comment]
KUBE_ENABLE_DAEMONSETS should be gone

**Pull request closed without merging**